<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="comPortLabel">
      <property name="text">
       <string>COM Port</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="distanceLabel">
      <property name="text">
       <string>Distance (m)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="distanceInput"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="outputLabel">
      <property name="text">
       <string>Output File Location</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <layout class="QHBoxLayout" name="pathLayout">
      <item>
       <widget class="QLineEdit" name="pathEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="browseButton">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="notesLabel">
      <property name="text">
       <string>Notes</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPlainTextEdit" name="notesEdit"/>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="goButton">
      <property name="text">
       <string>GO!</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="rescanButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Rescan</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comPortCombo"/>
      </item>
     </layout>
    </item>
    <item row="6" column="1">
     <widget class="QProgressBar" name="progressBar">
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="uuidEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="uuidLabel">
      <property name="text">
       <string>Beacon UUID</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
